### ISSUE TITLE
Awesome work ! 

### ISSUE THREAD

**Issue by NeuralDEEPnet** [malebuffy/Kobra-Max-Cobra-Kai-UI-and-Firmware#8]
Sonic pad failef form me. this is the best Update, for me almost happy as with Klipper

**Comment by malebuffy** [comment]
I could never use the USB on the Kobra Max so this was a no go for me. That's why  i modified the firmware.

**Comment by NeuralDEEPnet** [comment]
Right, however extruder stops mid air many times now. Slicing in Cura. Stock firm works fine

**Comment by malebuffy** [comment]
Did you disable stealthchop of E stepper like in the instructions?

Στις Σάβ 7 Οκτ 2023, 21:51 ο χρήστης NeuralDEEPnet ***@***.***>
έγραψε:

**Comment by NeuralDEEPnet** [comment]
NOpe
